jax/_src/pallas/mosaic/pipeline.py

```diff
@@ -287,11 +287,11 @@ def init_slots(self):
     """Initialize slot indices."""
     raise NotImplementedError()
 
-  def swap_slots(self, predicate: bool = True) -> "BufferedRefBase":
+  def swap_slots(self, predicate: bool = True) -> BufferedRefBase:
```

Comment:
FYI: I've seen code like this cause pytype failures despite the fact that it should be valid in 3.11